pkg/logger/logger.go

```diff
@@ -100,6 +100,7 @@ type OptionsLogger struct {
 	format          string
 	level           string
 	timestampFormat string
+	outputPaths     []string
```

Comment:
@senojj if we want to expose this to end users we have to introduce a new server flag